api/controllers/ConfigController.js

```diff
@@ -25,7 +25,48 @@
  * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
  */
 
+/* globals Config */
+/* globals ConfigService */
+
 module.exports = {
 	
+  create: function(req, res) {
+
+    let key = req.body.data.attributes.key;
+    let value = req.body.data.attributes.value;
+
+    return Config.findOrCreate({
```

Comment:
I'm pretty sure `ConfigService.set` does the `findOrCreate` for you. Can you test please ?